### ISSUE TITLE
[Model] HistGradientBoostingClassifier 기반 예측 모델

### ISSUE THREAD

**Issue by Ocean-2930** [SKN26-2nd-1st/2nd_project#34]
## 🤖 Model

- HistGradientBoostingClassifier
- GridSearchCV 활용한 최적 파라미터 탐색

---

## ⚙️ Training Setup

- Dataset: df_encoded_final.csv
- Feature: 모든 컬럼
- Hyperparameter: 최적 parameter 탐색을 위해서 여러 번 Grid Search 반복

---

## 📈 Expected Improvement

score 최고 성능 모델 탐색
accuracy score 0.8 이상
